src/api.ts

```diff
@@ -1,7 +1,7 @@
 import { MovieData } from './types/MovieData';
 import { ResponseError } from './types/ReponseError';
 
-const API_URL = 'https://www.omdbapi.com/?apikey=your-key';
+const API_URL = ' http://www.omdbapi.com/?i=tt3896198&apikey=9b92b38f';
```

Comment:
```suggestion
const API_URL = 'https://www.omdbapi.com/?apikey=9b92b38f';
```

It should help with failted requests
![image](https://github.com/mate-academy/react_movies-list-fetch-movies/assets/80700247/02955d50-223c-4a3c-ac90-6d181f5072c4)